### PR TITLE
Add embedding worker

### PR DESCRIPTION
**Pull request by latekvo**
This PR adds a perpetually embedding worker, as well as a separate, wrapped FAISS vector database.

### CODE REVIEW

**Comment by latekvo** [comment]
local black reformatter does not show up any issues, there is an issue with this workflow

File: workers/embedder.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        document = url_object["text"]
+
+        db_full_name = utils.gen_vec_db_full_name("embeddings", embed_model_name)
+        db_embeddings.db_add_text_batch(document, db_full_name)
+        db_url_pool.db_set_url_embedded(url_object["uuid"], embed_model_name)
+
+    print(f"{Fore.CYAN}Document vectorization completed.{Fore.RESET}")
+
+
+processing_iteration()
```
Comment (SOSNE) [inline workers/embedder.py]: I think this function should not be called here because it is called in a while loop.

**Review by SOSNE — COMMENTED** [review]
Added basic review.

**Review by SOSNE — APPROVED** [review]
Ship It 🌮

**Pull request merged**